src/cocoa/toga_cocoa/widgets/optioncontainer.py

```diff
@@ -49,16 +54,30 @@ def add_content(self, label, widget):
         self.native.addTabViewItem(item)
 
     def remove_content(self, index):
-        tabview = self.native.tabViewItemAtIndex(index)
-        self.native.removeTabViewItem(tabview)
+        try:
+            tabview = self.native.tabViewItemAtIndex(index)
+            self.native.removeTabViewItem(tabview)
+        except Exception as e:
+            print(e)
```

Comment:
I left this print, sorry.

Actually, in a particular case, this print it's showed: 
```
2020-04-24 18:41:44.146 python[96930:2120725] *** Assertion failure in -[NSTabView removeTabViewItem:], /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1894.40.150/AppKit.subproj/NSTabView.m:849
```
It's not an exception, so I can't handle it. 
To reproduce:
1) open example and disable option 2 and option 3
3) remove option 1 
3) try remove option 2 and that print appears. I think it's because cocoa want to make focus in an enabled tab but we don't have any tab enabled.

Reply:
Right - but that situation shouldn't exist, because step (2) should be illegal. There's nowhere to "land" after the removal, so that removal should raise an error.